### PR TITLE
Add rules for eslinter to improve linting quality 

### PR DESCRIPTION
**Pull request by denis-tingaikin**
Signed-off-by: denis-tingajkin <denis.tingajkin@xored.com>

### CODE REVIEW

File: server/server/src/__tests__/server.test.ts [hunk 1]
```diff
@@ -115,7 +115,7 @@ describe('server', () => {
   })
 
   test('should connect to server', async () => {
-    expect(conn.readyState).toEqual(1)
+    expect(await Promise.resolve(conn.readyState)).toEqual(WebSocket.OPEN)
```
Comment (haiodo) [inline server/server/src/__tests__/server.test.ts:118]: Why we need a promise here?

Comment (denis-tingaikin) [reply server/server/src/__tests__/server.test.ts:118]: Because async requires await.

Comment (z2sx) [reply server/server/src/__tests__/server.test.ts:118]: I believe we need to remove `async` here

Comment (denis-tingaikin) [reply server/server/src/__tests__/server.test.ts]: @haiodo , @z2sx  Thanks, fixed!

**Comment by z2sx** [comment]
Please add yourself to AUTHORS

**Comment by denis-tingaikin** [comment]
@z2sx OK, done!

**Comment by denis-tingaikin** [comment]
PR has updated, all checks passed,
Please have a look when you'll have a possible :)

**Pull request merged**